packages/extension-chrome/__tests__/services/ownership.ts

```diff
@@ -0,0 +1,296 @@
+import { createMockBackend, createMockKeystoreService, mockAddressInfos } from './utils';
+import { errors } from '@nexus-wallet/utils';
+import { Backend } from '../../src/services/backend/backend';
+import { DefaultAddressStorage } from '../../src/services/backend/addressStorage';
+import { createFullOwnershipService } from '../../src/services/fullOwnership';
+import { NotificationService } from '@nexus-wallet/types/lib';
+import { Cell, Transaction } from '@ckb-lumos/base';
+
+const mockNotificationService: NotificationService = {
+  requestSignTransaction: function (): Promise<{ password: string }> {
+    return Promise.resolve({ password: '123456' });
+  },
+  requestSignData: function (): Promise<{ password: string }> {
+    return Promise.resolve({ password: '123456' });
+  },
+  requestGrant: function (): Promise<void> {
+    errors.unimplemented();
+  },
+};
+
+describe('usedLocks and unusedLocks in ownership', () => {
```

Comment:
```suggestion
describe('ownership#getUsedLocks', () => {
```

only the `ownership.getUsedLocks` is found in this describe scope, so where can I find cases for `getUnusedLocks`